packages/dnb-eufemia/src/components/button/__tests__/Button.test.js

```diff
@@ -48,218 +49,241 @@ describe('Button component', () => {
-    expect(Comp.find('.dnb-button--size-medium').exists()).toBe(true)
+    render(<Component icon="question" />)
+    const button = document.querySelector('button')
 
+    // size "medium" and has icon
+    expect(button.classList.contains('dnb-button--size-medium')).toBe(true)
     // has icon class, but not has text
-    expect(Comp.find('.dnb-button--has-icon').exists()).toBe(true)
-    expect(Comp.find('.dnb-button--has-text').exists()).toBe(false)
+    expect(button.classList.contains('dnb-button--has-icon')).toBe(true)
+    expect(button.classList.contains('dnb-button--has-text')).toBe(false)
   })
 
   it('has size set to medium when button size is default', () => {
-    const Comp = mount(<Component icon="question" size="default" />)
-    expect(Comp.find('.dnb-button--icon-size-medium').exists()).toBe(true)
-    expect(Comp.find('.dnb-icon--medium').exists()).toBe(true)
+    render(<Component icon="question" size="default" />)
+    const button = document.querySelector('button')
+    const icon = document.querySelector('.dnb-icon')
+    expect(button.classList.contains('dnb-button--icon-size-medium')).toBe(
+      true
+    )
+    expect(icon.classList.contains('dnb-icon--medium')).toBe(true)
   })
 
   it('has medium icon if button size is large', () => {
-    const Comp = mount(
-      <Component text="Button" size="large" icon="question" />
-    )
-
+    render(<Component text="Button" size="large" icon="question" />)
+    const button = document.querySelector('button')
+    const icon = document.querySelector('.dnb-icon')
     // size "large
-    expect(Comp.find('.dnb-button--size-large').exists()).toBe(true)
-    expect(Comp.find('.dnb-icon--default').exists()).toBe(true)
+    expect(button.classList.contains('dnb-button--size-large')).toBe(true)
+    expect(icon.classList.contains('dnb-icon--default')).toBe(true)
   })
 
   it('has to have a bounding tag if property is set', () => {
-    const Comp = mount(<Component bounding={true} />)
-    expect(Comp.find('.dnb-button__bounding').exists()).toBe(true)
+    render(<Component bounding={true} />)
+    const bounding = document.querySelector('.dnb-button__bounding')
+    expect(bounding !== null).toBe(true)
   })
 
   it('has a anchor tag', () => {
-    const Comp = mount(
-      <Component {...props} href="https://url" icon={null} />
-    )
-    expect(Comp.find('a').exists()).toBe(true)
-    expect(Comp.find('svg').exists()).toBe(false)
+    render(<Component {...props} href="https://url" icon={null} />)
+    expect(document.querySelector('a') !== null).toBe(true)
+    expect(document.querySelector('svg') !== null).toBe(false)
   })
 
   it('has a anchor tag and includes a launch icon', () => {
-    const Comp = mount(
+    render(
       <Component
         {...props}
         href="https://url"
         target="_blank"
         icon={null}
       />
     )
-    expect(Comp.find('svg').exists()).toBe(true)
+    expect(document.querySelector('svg') !== undefined).toBe(true)
```

Comment:
```suggestion
    expect(document.querySelector('svg')).toBeTruthy()
```

Is probably easier to "read".